src/core/KinematicArea2D/KinematicArea2D.cs

```diff
@@ -3,11 +3,11 @@
 
 public partial class KinematicArea2D : Area2D
 {
-    [Export]
```

Comment:
Looks like your code editor uses tabs instead of spaces by default. This should be something you can configure. Can you remove these files with the changed spacing?